include/constants/battle.h

```diff
@@ -304,6 +304,9 @@
 #define B_WEATHER_SNOW_TEMPORARY      (1 << 12)
 #define B_WEATHER_SNOW_PERMANENT      (1 << 13)
 #define B_WEATHER_SNOW                (B_WEATHER_SNOW_TEMPORARY | B_WEATHER_SNOW_PERMANENT)
+#define B_WEATHER_FOG_TEMPORARY       (1 << 14)
```

Comment:
Temporary Fog doesn't exist in vanilla, since there's no way to set up Fog weather in battle, besides overworld weather.

Reply:
Its not used. I added it just to be consistent with the other weathers, and if someone wants to add it to a move.

Reply:
Note to anybody actually adding fog as a move: Don't.